TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>We should add a link to the reference(s) here. We should also be more explicit about what each type is (i.e. SAP_flux is Simple Aperture Photometry, "raw data", just a sum of the pixels in the aperture)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>added link and SAP explanation</p>